karaf/init/pom.xml

```diff
@@ -0,0 +1,79 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+        <artifactId>brooklyn-tosca-karaf</artifactId>
+        <version>0.10.0-SNAPSHOT</version> <!-- BROOKLYN_TOSCA_VERSION -->
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>brooklyn-tosca-karaf-init</artifactId>
+    <packaging>bundle</packaging>
+    <name>Brooklyn TOSCA Karaf OSGI Initialization</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+            <artifactId>brooklyn-tosca-karaf-deps-brooklyn</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+            <artifactId>_brooklyn-tosca-karaf-patches</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>io.cloudsoft.brooklyn.tosca</groupId>
+            <artifactId>brooklyn-tosca-transformer</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+<!--
+                <version>3.2.0</version>
+-->
+                <extensions>true</extensions>
+                <configuration>
+                    <supportedProjectTypes>
+                        <!--
+                         Very strange that we have to include this section. The project is quite happy to build
```

Comment:
Strange and scary. Now that we're changing `karaf/deps-brooklyn/pom.xml` to be `<packaging>bundle</packaging>`, we should watch out for this. But it seemed to be fine in deps-brooklyn without this.